src/coverlet.collector/DataCollector/CoverletCoverageDataCollector.cs

```diff
@@ -0,0 +1,179 @@
+﻿namespace Coverlet.Collector.DataCollector
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Xml;
+    using coverlet.collector.DataCollector;
+    using Coverlet.Collector.Utilities;
+    using Coverlet.Collector.Utilities.Interfaces;
+    using Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection;
+
+    /// <summary>
+    /// Coverlet coverage out-proc data collector.
+    /// </summary>
+    [DataCollectorTypeUri(CoverletConstants.DefaultUri)]
+    [DataCollectorFriendlyName(CoverletConstants.FriendlyName)]
+    public class CoverletCoverageCollector : DataCollector
+    {
+        private readonly TestPlatformEqtTrace eqtTrace;
```

Comment:
Please prefix instance field with `_eqtTrace`

Reply:
Agree, style should be `_` prefix instead of using `this.`.

Reply:
Done.